File: benchmark/results/python-invocation-script/run-003/b-baseline/output/metaengine_demo/billing/value_objects/invoice_line.py
"""InvoiceLine value object for the billing domain."""
from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class InvoiceLine:
    """Immutable value object representing a single invoice line."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str

File: benchmark/results/java-invocation-script/run-005/a-mcp/output/billing/value_objects/InvoiceLine.java
package com.metaengine.demo.billing.value_objects;

import java.time.OffsetDateTime;

// InvoiceLine value object.

public class InvoiceLine {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public InvoiceLine(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}
